commerce/domain/shipping/splitshipment/SplitShipmentUnitTest.cls

```diff
@@ -0,0 +1,303 @@
+/**
+ * @description Sample unit test for SplitShipmentSample and SplitShipmentCallsSuper.
+ */
+@IsTest
+global class SplitShipmentUnitTest {
+
+    private static final MockDefaultSplitShipmentService mockService = new MockDefaultSplitShipmentService();
+    private static final SplitShipmentCallsSuper splitShipmentCallsSuper = new SplitShipmentCallsSuper(mockService);
+    private static final SplitShipmentSample splitShipmentSample = new SplitShipmentSample();
+
+    @IsTest
+    public static void splitShipmentApexThatCallsSuperShouldInvokeDefaultSplitShipmentService() {
+        // Arrange
+        CartExtension.Cart cart = CartExtension.CartTestUtil.createCart();
+
+        CartExtension.ItemArrange itemArrange = new CartExtension.ItemArrange.Builder()
+                .withProductId('01txx0000006iP2AAI')
+                .withQuantity(Decimal.valueOf(5))
+                .withCartItemId(cart.getCartItems().get(0).getId())
+                .withDeliveryGroupId(cart.getCartItems().get(0).getCartDeliveryGroup().getId())
+                .withDeliveryGroupName(CartExtension.CartTestUtil.DELIVERYGROUP_NAME)
+                .withDeliverToCity('Foxboro')
+                .withDeliverToCountry('USA')
+                .withDeliverFromFirstName('Tom')
+                .withDeliverFromLastName('Brady')
+                .withDeliverToName('Bill Belichick')
+                .withDeliverToPostalCode('02035')
+                .withDeliverToState('MA')
+                .withDeliverToStreet('2 Patriot Place')
+                .withDeliverToCompanyName('New England Patriots')
+                .withDeliveryAddressId('8lW456789012679AAA')
```

Comment:
Is this a possible scenario when ItemArrangment has both DeliveryAddress and DeliveryAddressId?